clicache/test2/Tests2.cs

```diff
@@ -123,7 +123,24 @@ public void RegisterPdxType()
       Assert.False(_pdxDelegate2Called);
     }
 
-    private class DummyPdxSerializer : IPdxSerializer
+    [Fact]
+    public void SetSniProxy()
+    {
+        PoolFactory poolFactory = _cacheOne.GetPoolFactory()
+                .AddLocator("localhost", 10334);
+
+        poolFactory.SetSniProxy("haproxy", 7777);
```

Comment:
I'd like to see this right after `AddLocator` in the typical builder pattern look...

Reply:
Good catch.